### DISCUSSION TITLE
Flux toast improvements

### DISCUSSION THREAD

**Discussion by jsandfordhughescoop** [livewire/flux#367] [Feature Requests]
Hey, 

I have made a couple of changes to the Toast component for our internal team to use, it adds variants and a fluid API.

It would be great to have these in the core, maybe also a position property.

Code is below in a gist. 

https://gist.github.com/jsandfordhughescoop/7ab014ca213edbe0b69bf34256e98a75

**Comment by RiadhKHEDHIRI** [comment]
Are multiple toasts stackable? I fired two toasts, and only the last one was displayed.

**Comment by dudelisius** [comment]
20 hours ago the latest version 1.0.9 was release and this supports [variants](https://fluxui.dev/components/toast#variants) and [positions](https://fluxui.dev/components/toast#positions).
